### PR TITLE
feat: added question page

### PR DESCRIPTION
**Pull request by raman1999**
image

### CODE REVIEW

File: screens/questions/question.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/stylesheets/components.css">
+    <link rel="stylesheet" href="https://rocket-ui-library.netlify.app/landing.css">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="./question.css">
```
Comment (shuntzhere) [inline screens/questions/question.html]: Import all the CSS files to one main file and use that across.

File: screens/questions/question.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  width: 21rem;
+  max-width: 30rem;
+}
+.question-container {
+  margin-top: 1rem;
+}
+
+.question-btn-container {
+  display: flex;
+  justify-content: space-between;
```
Comment (shuntzhere) [inline screens/questions/question.css:20]: You can use these as utility classes and import them from the component library. This way you won't be repeating code.

**Review by shuntzhere — COMMENTED** [review]
Suggested some minor changes. The UI looks really good.

File: screens/questions/question.html (hunk 1 above)
Comment (raman1999) [reply screens/questions/question.html]: Ok,thanks

File: screens/questions/question.css (hunk 2 above)
Comment (raman1999) [reply screens/questions/question.css:20]: I have used utility classes mostly, but this is specifically used for this one only and not for other components,

File: screens/questions/question.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    <!-- Navbar -->
+    <nav class="nav">
+        <div class="nav-logo">
+            <span class="nav-header txt-theme-8">QuizMaster</span>
+        </div>
+        <span id="btn-nav" class="nav-btn"><i class="fas fa-chevron-circle-down fa-lg"></i></span>
+        <ul id="list-nav" class="nav-list nav-list-hide">
+            <li class="list-item txt-bold">Home</li>
+            <li class="list-item  txt-bold">Logout</li>
+
```
Comment (nishasen) [inline screens/questions/question.html]: Remove the extra space

**Review by nishasen — COMMENTED** [review]
The code looks clean and easily understandable. good work

**Review by nishasen — APPROVED** [review]

**Pull request merged**